.github/workflows/superset-frontend.yml

```diff
@@ -25,18 +31,3 @@ jobs:
       working-directory: ./superset-frontend
       run: |
         bash <(curl -s https://codecov.io/bash) -cF javascript
-
-  docs:
```

Comment:
Was this even working? Looks like it was running from `./superset-frontend`.

Reply:
Nope... but I hope it will now.